spec/decorators/concrete_slot_decorator_spec.rb

```diff
@@ -44,13 +44,11 @@
         before do
           switch_on :nomis_staff_prisoner_availability_enabled
         end
-
         context 'with a closed restriction' do
           let(:slot_errors) { [Nomis::Restriction::CLOSED_NAME] }
 
           it 'renders the checkbox with errors' do
             expect(html_fragment).to have_css('label.date-box__label.date-box--error')
-            expect(html_fragment).to have_css('span.date-box__number', text: '1')
```

Comment:
Why are these expectations deleted? What was in the text with value `1` ?

Reply:
We are no longer showing the numbers 1,2,3 in the date boxes